frontend/static/dashboard.html

```diff
@@ -0,0 +1,23 @@
+
+<div class="dashboard-container" style="width: 800px;"> <!-- Fixed width of 800px for the dashboard container -->
+    <h2 data-i18n="navbar.dashboard" style="margin-bottom: 40px;"></h2>
+    
+    <h3 data-i18n="dash.player-results"></h3>
+    <canvas id="gameResultsChart" style="width: 800px; margin-bottom: 40px;"></canvas> <!-- Add margin-bottom -->
+
+    <!-- New section for victory/loss pie chart -->
+    <h3 data-i18n="dash.player-distri"></h3>
+    <canvas id="victoryLossChart" style="width: 800px; margin-bottom: 40px;"></canvas> <!-- Add margin-bottom -->
+
+    <h3 data-i18n="dash.game-duration"></h3>
+    <canvas id="gameDurationChart" style="width: 800px; margin-bottom: 40px;"></canvas> <!-- Add margin-bottom -->
+
+    <h3 data-i18n="dash.game-intensity"></h3>
+    <canvas id="gameIntensityChart" style="width: 800px; margin-bottom: 40px;"></canvas> <!-- Add margin-bottom -->
```

Comment:
Add the styles in the style.css
Use Bootstrap as much as possible. 
For margin-bottom, change to Bootstrap